lib/src/main/kotlin/io/wavebeans/lib/stream/window/WindowFunction.kt

```diff
@@ -0,0 +1,66 @@
+package io.wavebeans.lib.stream.window
+
+import io.wavebeans.lib.*
+import io.wavebeans.lib.stream.map
+import kotlin.math.PI
+import kotlin.math.abs
+import kotlin.math.cos
+
+class MapWindowFn<T : Any>(initParameters: FnInitParameters) : Fn<Window<T>, Window<T>>(initParameters) {
```

Comment:
add KDocs:
Map function that applies the specified window function on the windowed stream. The function is applied if `w` is window value and `f` is window function, then the result is `w * f`, which is multiplication of corresponding elements.